app.js

```diff
@@ -1,17 +1,17 @@
-angular.module('postApp', ['ui.router', 'ngResource'])
-  .config( postConfig );
+angular.module('bookApp', ['ui.router', 'ngResource'])
+  .config( bookConfig );
 
-  function postConfig ( $locationProvider, $stateProvider, $urlRouterProvider ) {
-      // return to post-index if bad route request
+  function bookConfig ( $locationProvider, $stateProvider, $urlRouterProvider ) {
```

Comment:
`bookConfig.$inject = ['$locationProvider', '$stateProvider', '$urlRouterProvider']`

https://github.com/sf-wdi-labs/angular_style_guide#use-inject-vs-inline-annotation-for-dependency-injection